public/local.js

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable camelcase */
```

Comment:
I know that this is not the change in your new code, but my though is to why don't we just remove the two lines and fix the code instead of depressing?

Reply:
because this is a "config" from outside the module. We probably should find a better way to do it. It's a very Old javascript approach because I didn't know a better way.

Reply:
That's fine. We can discuss to due with this later.